src/mem/heap/heap_simple.c

```diff
@@ -134,6 +134,48 @@ void *malloc(size_t size) {
 	return 0;
 }
 
+void *memalign(size_t boundary, size_t size) {
```

Comment:
Now you have two functions - `memalign` and `malloc` which do almost the same things. We should use memalign in malloc instead.

Reply:
Done :).

Reply:
Yes, I like how it looks now :) Only small comment about `memalign(1, ` below.